src/sinks/kafka/config.rs

```diff
@@ -19,28 +19,66 @@ use crate::{
 
 pub(crate) const QUEUED_MIN_MESSAGES: u64 = 100000;
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+/// Configuration for the `kafka` sink.
+#[configurable_component(sink)]
+#[derive(Clone, Debug)]
 #[serde(deny_unknown_fields)]
-pub(crate) struct KafkaSinkConfig {
+pub struct KafkaSinkConfig {
+    /// A comma-separated list of the initial Kafka brokers to connect to.
+    ///
+    /// Each value must be in the form of `<host>` or `<host>:<port>`, and separated by a comma.
```

Comment:
I'd thought `port` was required, does rdkafka default the port if it's not provided?

Reply:
Our current documentation for the `kafka` sink does imply that both host and port are required, but the `librdkafka` documentation [says otherwise.](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md) (search for `bootstrap.servers`)

Reply:
Word 👍